### PR TITLE
Prometheus timerange bug #27

### LINKED ISSUE #27: Prometheus: timerange bug
When trying to export data out of existing timerange - it returns existing data outside timerange
Seems like `from` and `to` params of `getResults` don't work

### PR DESCRIPTION
**Pull request by amper43**
- fixes of from/to args processing

fixes #27

### CODE REVIEW

File: src/metrics/prometheus_metric.ts [hunk 1]
```diff
@@ -4,18 +4,19 @@ const QUERY_TIME_REGEX = /\&start=[^\&]*\&end=[^\&]*\&/;
...
   constructor(datasource: Datasource, targets: any[], id?: MetricId) {
     super(datasource, targets, id);
-
-    this._queryParts = datasource.type.split(QUERY_TIME_REGEX);
   }
 
   getQuery(from: number, to: number, limit: number, offset: number): MetricQuery {
-    let q = `${this._queryParts[0]}&start=${from}&end=${to}&${this._queryParts[1]}`;
+    let url = this.datasource.url;
+    from = Math.floor(from/1000); //prometheus uses seconds for timestamp
```
Comment (jonyrock) [inline src/metrics/prometheus_metric.ts]:
```suggestion
    from = Math.floor(from / 1000); // prometheus uses seconds for timestamp
```

File: src/metrics/prometheus_metric.ts [hunk 2]
```diff
@@ -4,18 +4,19 @@ const QUERY_TIME_REGEX = /\&start=[^\&]*\&end=[^\&]*\&/;
...
+    to = Math.floor(to/1000);
```
Comment (jonyrock) [inline src/metrics/prometheus_metric.ts]:
```suggestion
    to = Math.floor(to / 1000);
```

**Comment by jonyrock** [comment]
@amper43 lets add a test for this bug, please

**Comment by amper43** [comment]
@jonyrock test added

**Pull request merged**